src/core/server/workspaces/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -96,6 +96,16 @@ export class WorkspaceSavedObjectsClientWrapper {
       objects: Array<SavedObjectsBulkCreateObject<T>>,
       options: SavedObjectsCreateOptions = {}
     ): Promise<SavedObjectsBulkResponse<T>> => {
+      if (options.workspaces) {
+        options.workspaces = options.workspaces.filter(
+          async (workspaceId) =>
+            await this.permissionControl.validate(
+              workspaceId,
+              WorkspacePermissionMode.Admin,
```

Comment:
```suggestion
              [WorkspacePermissionMode.Write, WorkspacePermissionMode.Management],
```

Should be like this I think?